CHANGELOG.md

```diff
@@ -2,283 +2,520 @@
 
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
-and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```

Comment:
It doesn't as we use `-rc<x>` for release candidates. Semver uses `-rc.<x>`

Reply:
Done https://github.com/3mdeb/meta-rte/compare/59973da3165f4b6227a4a5eb9279c0cb3b63d074..c3d9963d89997d076211040aeac3e5fba9cab212#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL6